pkg/operator/controllers/machine/const.go

```diff
@@ -0,0 +1,9 @@
+package machine
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+const (
+	// replaces pkg/operator/controllers/checker/const.go
+	machineSetsNamespace = "openshift-machine-api"
```

Comment:
@cadenmarchese your lint-go tests are failing because you need to delete machineSetsNamespace in `pkg/operator/controllers/checker/const.go`.  It's not being used anymore because it is now here.

Reply:
Fixed, thanks!